README.md

```diff
@@ -206,6 +206,9 @@ The import package can be used as a command-line tool and supports a several dif
 - (Required) `--source-folder` The source folder or file to scan for files/archives to process.
 - (Optional) `--output-folder` The folder to save the output. The specified folder must exists. Defaults to `./`.
 - (Optional) `--output-name` The name, including extension, of the output file. Defaults to `output.csv`.
+- (Optional) `--text-name` The name to text field, case this field has a different name.
```

Comment:
Could you change the CLI tool params to --text-field-key, --date-field-key and --id-field-key, please? I know it's longer for the user but I think it would be clearer.
Also, just for the syntax of the param description:
"The key of the text field, if not *text*"
"The key of the date field, if not *created_at*"
"The key of the id field, if not *id*"

Reply:
yes!